Objects/Objects/Utils/MeshTriangulationHelper.cs

```diff
@@ -242,22 +242,16 @@ public Vector3(double x, double y, double z)
 
     public static readonly Vector3 Zero = new(0, 0, 0);
 
-    public static Vector3 operator +(Vector3 a, Vector3 b)
-    {
-      return new Vector3(a.x + b.x, a.y + b.y, a.z + b.z);
-    }
+    public static Vector3 operator +(Vector3 a, Vector3 b) => new(a.x + b.x, a.y + b.y, a.z + b.z);
 
-    public static Vector3 operator -(Vector3 a, Vector3 b)
-    {
-      return new Vector3(a.x - b.x, a.y - b.y, a.z - b.z);
-    }
+    public static Vector3 operator -(Vector3 a, Vector3 b) => new(a.x - b.x, a.y - b.y, a.z - b.z);
 
-    public double Dot(Vector3 v)
+    public readonly double Dot(Vector3 v)
```

Comment:
Nice!

Reply:
Actually, I this has made me think, we could probably look into making the entire struct `readonly`...
(tho, nothing more needed in this PR)